app/assets/javascripts/actions/task.js

```diff
@@ -0,0 +1,46 @@
+import actionTypes from './actionTypes';
+
+export const createTaskSuccess = (task, storyId) => ({
+  type: actionTypes.ADD_TASK,
+  task,
+  storyId
+});
+
+export const deleteTaskSuccess = (task, story) => ({
+  type: actionTypes.REMOVE_TASK,
+  task,
+  story
```

Comment:
Does this variable store the actual story or only the id? Because looks like you pass it as `storyId` here https://github.com/Codeminer42/cm42-central/pull/440/files#diff-4683d203f1fd881b51e0d93c7105d8fcR34. Can we rename it? Rember it's gonna impact the reducer as well.

Reply:
Ok!